specification/network/resource-manager/Microsoft.Network/stable/2020-03-01/privateEndpoint.json

```diff
@@ -567,6 +860,98 @@
         }
       },
       "description": "The information of an AvailablePrivateEndpointType."
+    },
+    "PrivateDnsZoneGroup": {
+      "properties": {
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Name of the resource that is unique within a resource group. This name can be used to access the resource."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource type."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/PrivateDnsZoneGroupPropertiesFormat",
+          "description": "Properties of the private dns zone group."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/SubResource"
+        }
+      ],
+      "description": "Private dns zone group resource."
+    },
+    "PrivateDnsZoneGroupPropertiesFormat": {
+      "properties": {
+        "provisioningState": {
+          "readOnly": true,
+          "$ref": "./network.json#/definitions/ProvisioningState",
+          "description": "The provisioning state of the private dns zone group resource."
+        },
+        "privateDnsZoneConfigs": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrivateDnsZoneConfig"
+          },
+          "description": "A collection of private dns zone configurations of the private dns zone group."
+        }
+      },
+      "description": "Properties of the private dns zone group."
+    },
+    "PrivateDnsZoneConfig": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/PrivateDnsZoneConfigProperties",
+          "description": "Properties of the private dns zone configuration."
+        }
+      },
+      "description": "PrivateDnsZoneConfig resource."
+    },
+    "PrivateDnsZoneConfigProperties": {
+      "properties": {
+        "privateDnsZoneId": {
+          "type": "string",
+          "description": "The resource id of the private dns zone."
+        },
+        "recordSets": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RecordSet"
+          },
+          "description": "A collection of information regarding a recordSet, holding information to identify private resources."
+        }
+      },
+      "description": "Properties of the private dns zone configuration resource."
+    },
+    "RecordSet": {
+      "properties": {
+        "recordType": {
+          "type": "string",
+          "description": "Resource record type."
+        },
+        "recordSetName": {
+          "type": "string",
+          "description": "Recordset name."
+        },
+        "fqdn": {
+          "type": "string",
+          "description": "Fqdn that resolves for private endpoint ip address."
+        },
+        "ipAddresses": {
+          "type": "array",
```

Comment:
also add ttl here

Reply:
And also provisioningState

Reply:
what is ttl? and I think pilor has mentioned in a previous conversation provisioning state is for child resources?

Reply:
Time to live. See these contracts: https://msazure.visualstudio.com/One/_git/Networking-nrp?path=%2Fsrc%2Fsources%2FFrontend%2FFrontendContract%2FCsm.Public%2FPrivateDnsZoneConfigRecordSet.cs&_a=contents&version=GBmaster

Reply:
will this be readonly?